CRM/Admin/Form/ScheduleReminders.php

```diff
@@ -158,6 +158,11 @@ public function buildQuickForm() {
     $this->add('number', 'start_action_offset', ts('When'), ['class' => 'six', 'min' => 0]);
     $this->addRule('start_action_offset', ts('Value should be a positive number'), 'positiveInteger');
 
+    $date = $this->add('datepicker', 'created_date', ts('Created on'), [], FALSE);
```

Comment:
Although 'Created on' has a nice informal feel to it, I think I'd prefer the field label to match the field name, so let's change to 'Created date'.